utils/storybook/CanvasProviderDecorator.tsx

```diff
@@ -5,11 +5,21 @@ import {
   createCanvasTheme,
 } from '@workday/canvas-kit-labs-react-core';
 import {object} from '@storybook/addon-knobs';
+import {makeDecorator} from '@storybook/addons';
 
 const label = 'theme';
 
-export default (storyFn: () => React.ReactNode) => (
-  <CanvasProvider theme={createCanvasTheme(object(label, defaultCanvasTheme))}>
-    {storyFn()}
-  </CanvasProvider>
-);
+export default makeDecorator({
+  name: 'withCanvasProviderDecorator',
+  parameterName: 'canvasProviderDecorator',
+  wrapper: (storyFn, context, {parameters}) => {
+    return (
+      <CanvasProvider
+        includeInputProvider={parameters ? parameters.disabled : undefined}
```

Comment:
We should change the name of this parameter to match the name of the `includeInputProvider` prop. If I use `.addParameters({ canvasProviderDecorator: { disabled: true }})`, I wouldn't expect the `CanvasProvider` to render at all.

You can also simplify this check:
```tsx
  wrapper: (storyFn, context, {parameters = {}}) => {
);	    return (
      <CanvasProvider
        includeInputProvider={parameters.includeInputProvider}
```
(note the default in the `parameters` instantiation)